### ISSUE TITLE
frio: Content background transparency is wrong

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#3824]
Hi,
since today (develop) the custom theme setting "Content background transparency" is wrong. I have to set the slider to transparency 100% to get no transparency.

**Comment by AlfredSK** [comment]
The behavior started with this PR #3821 .

**Comment by rabuzarus** [comment]
>  I have to set the slider to transparency 100% to get no transparency.
This behavior is the original behavior. Just as it works before the pull request. But there was a bug in presenting the correct position of the slider (Often the value was displayed at 20 while the correct value was 100).
With this PR another bug was fixed. Before the PR it wasn't it wasn't possible to set the value to 0. If someone did set the value it was automatically set to 100. Now it is possible to have a value of 0. This does explain why you don't have a background background-color. But I can't say why it was set to 0.

I guess the value 0 was saved some time before because the PR doesn't automatically change the user settings in the db.

**Comment by AlfredSK** [comment]
Yep. I saw the default value in the code. :-) It is indeed 100. I had to manually reconfigure all my accounts because they were all showing no content background all of a sudden. But that is not the point, actually. 
My point is the setting is named "transparency". Transparency 100% means something is completely "see-through". Transparency 0% means something is completely opaque. The slider (and the value) does exactly the opposite. 
A simple solution would be to rename it to "content background opacity". :-)

**Comment by rabuzarus** [comment]
Thank you, will do it :-)

**Comment by rabuzarus** [comment]
@AlfredSK 
I think this was fixed. Can we close it?

**Comment by AlfredSK** [comment]
I'll have a look in a couple of minutes. I'm a little busy at the moment with database stuff. :-)

**Comment by rabuzarus** [comment]
no hurry. this isn't important ;-)

**Comment by AlfredSK** [comment]
From my point of view it is still the wrong way round.
The slider is called "background transparency". If I set the transparency to 100% I get zero transparency. If I set the transparency to 0% I get full transparency. That's confusing.

Because I was unsure I checked the Gnome Terminal. There is a background transparency slider too. It works like expected. Slider to the left -> no transparency. Slider to the right -> full transparency. Exactly the opposite of how our slider works.

I know this is some kind of nitpicking. :-) When nobody else is complaining feel free to close the issue. 

My proposal is still to just rename the slider to "background **opacity**". That would solve the problem easily without breaking something.
